automatic/absolute-uninstaller/tools/chocolateyInstall.ps1

```diff
@@ -1,6 +1,19 @@
-﻿$packageName = '{{PackageName}}'
-$installerType = 'EXE'
-$url = '{{DownloadUrl}}'
-$silentArgs = "/S" # NSIS package
-$validExitCodes = @(0)
-Install-ChocolateyPackage "$packageName" "$installerType" "$silentArgs" "$url" -validExitCodes $validExitCodes
+﻿$packageName = 'absolute-uninstaller'
+$url = 'http://download.glarysoft.com/ausetup.exe'
+$checksum = '1a9146ac0eaa94b6389625e743cc17406073f4217b3a46df3799dc834fe8119a'
+$checksumType = 'sha256'
+
+$packageArgs = @{
+  packageName = $packageName
+  fileType = 'exe'
+  softwareName = 'Absolute Uninstaller'

```

Comment:
I believe it does, but I'll take another test run in the test environment just to be sure before I push the package source changes.